client/js/widgets/widget.js

```diff
@@ -15,6 +15,7 @@ export class Widget extends StateManaged {
     this.id = id;
     this.domElement = div;
     this.childArray = [];
+    this.propertiesUsedInCSS = [];
```

Comment:
Does `propertiesUsedInCSS` ever get checked when properties get updated to see if the stylesheet needs rebuilding. I don't see that code anywhere.

Reply:
It's appended to `cssProperties()`.